src/test/java/com/zerobank/pages/LoginPage.java

```diff
@@ -51,6 +51,10 @@ public void login(String username, String password){
 
     }
 
+    public void thisIsForTest(){
+        // this is commment for git
+    }
+
```

Comment:
Can you double check this part one more time?

Reply:
it looks great thank you